modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -140,6 +142,15 @@ def validate_form_526_title10_anticipated_separation_date!
       )
     end
 
+    def transform_form_526_unit_name!
+      reserves = form_attributes&.dig('serviceInformation', 'reservesNationalGuardService')
+      return if reserves.nil?
+
+      unit_name = reserves['unitName']
+      unit_name = unit_name.presence || ' '
+      reserves['unit_name'] = unit_name
+    end
+
```

Comment:
@stiehlrod This is a transform basically just for the docker container I believe.  Wondering if this would be better off in the `auto_established_claim` model where other transforms are?  I think we really only need this for when .to_internal gets called before sending to the docker container similar to something like the previous [unit phone transform](https://github.com/department-of-veterans-affairs/vets-api/pull/18837/files)

Reply:
You would also then be able to test the returns in `modules/claims_api/spec/models/auto_establish_claim_spec.rb` I beieve to check for any expected values like making sure it did transform into `" "`